src/types/server-types.js

```diff
@@ -68,6 +68,33 @@ export type LoginResponse = {
 // request payloads
 // ---------------------------------------------------------------------
 
+export type LoginCreatePayload = {
+  appId: string,
+  loginId: string, // base64
+  parentBox?: EdgeBox
+
+  // The creation payload can also include fields
+  // from any of these other types, so the server should try
+  // those cleaners one-by-one and incorporate the ones that work:
+  // ...KeyCreatePayload
+  // ...OtpPayload
+  // ...PasswordPayload
+  // ...Pin2EnablePayload
+  // ...Pin2DisablePayload
+  // ...Recovery2Payload
+  // ...SecretPayload
+}
+
+export type KeyCreatePayload = {
```

Comment:
Typo: Should have been `KeysCreatePayload` (plural), to match the URL that takes this payload.

Reply:
Fixup commit addresses this.